lib/pgBackRest/Common/Ini.pm

```diff
@@ -639,19 +680,17 @@ sub test
 }
 
 ####################################################################################################################################
-# boolTest
-#
-# Test a value to see if it equals the supplied test boolean value.  If no test value is given, tests that it is defined.
+# boolTest - test a boolean value, see test().
 ####################################################################################################################################
 sub boolTest
 {
-    my $self = shift;
-    my $strSection = shift;
-    my $strValue = shift;
-    my $strSubValue = shift;
-    my $bTest = shift;
-
-    return $self->test($strSection, $strValue, $strSubValue, defined($bTest) ? ($bTest ? INI_TRUE : INI_FALSE) : undef);
+    return shift->test(shift, shift, shift, defined($_[0]) ? (shift() ? INI_TRUE : INI_FALSE) : undef);
 }
 
+####################################################################################################################################
+# Properties.
+####################################################################################################################################
+sub modified {shift->{bModified}}                                   # Has the data been modified since last load/save?
```

Comment:
I don't see a call to modified anywhere - are you planning to use it in the next part of the patch?